src/schema/capital-project.ts

```diff
@@ -21,14 +21,16 @@ export const capitalProjectCategoryEnum = pgEnum("capital_project_category", [
 export const capitalProject = pgTable(
   "capital_project",
   {
-    managingCode: char("managing_code", { length: 3 }).references(
-      () => managingCode.id,
-    ),
-    id: text("id"),
-    managingAgency: text("managing_agency").references(() => agency.initials),
-    description: text("description"),
-    minDate: date("min_date"),
-    maxDate: date("max_date"),
+    managingCode: char("managing_code", { length: 3 })
+      .references(() => managingCode.id)
+      .notNull(),
+    id: text("id").notNull(),
+    managingAgency: text("managing_agency")
+      .references(() => agency.initials)
+      .notNull(),
+    description: text("description").notNull(),
+    minDate: date("min_date").notNull(),
+    maxDate: date("max_date").notNull(),
```

Comment:
updates to the drizzle table definitions require database migrations. Let's combine these into the same migration file as the upcoming spatial index